### PR TITLE
Fix paste multiline text issue(T1130660)

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: test/unit/core/editor.js [hunk 1]
```diff
@@ -897,6 +897,11 @@ describe('Editor', function () {
       expect(editor.getHTML(0, 11)).toEqual('&lt;b&gt;Test&lt;/b&gt;');
     });
 
+    it('multiline html', function () {
```
Comment (ksercs) [inline test/unit/core/editor.js:900]: What do we test here? "last line break should not be parsed on editor initialization"?
Please, make test name more strict - i didn't catch which scenario is tested here

File: test/unit/modules/clipboard.js [hunk 2]
```diff
@@ -31,6 +31,26 @@ describe('Clipboard', function () {
...
+          clipboardData: {
+            ...this.clipboardData,
+            getData: (type) => (type === 'text/html' ? '1<br>' : '1\n'),
+          },
+        };
+
+        this.quill.clipboard.onCapturePaste(captureData);
+        setTimeout(() => {
+          expect(this.quill.root).toEqualHTML(
+            '<p>1</p><p>8</p>',
```
Comment (ksercs) [inline test/unit/modules/clipboard.js:48]: If line break is not pasted, a result will be like `<p>18</p>`?

File: test/unit/modules/clipboard.js [hunk 3]
```diff
@@ -31,6 +31,26 @@ describe('Clipboard', function () {
         }, 2);
       });
 
+      it('paste multiline text', function (done) {
```
Comment (ksercs) [inline test/unit/modules/clipboard.js:34]:
```suggestion
      it('last line break should not be cut on a multiline text paste', function (done) {
```

File: test/unit/modules/clipboard.js (hunk 2 above)
Comment (alexanderPolosatov) [reply test/unit/modules/clipboard.js:48]: yes

File: modules/clipboard.js [hunk 4]
```diff
@@ -223,15 +228,15 @@ class Clipboard extends Module {
     }
   }
 
-  onCopy(range) {
-    const text = this.quill.getText(range);
-    const html = this.quill.getSemanticHTML(range);
+  onCopy({ index, length }) {
+    const text = this.quill.getText(index, length);
```
[the remark this answers is not in the document]
Comment (alexanderPolosatov) [reply modules/clipboard.js:232]: getSemanticHTML and getText accepts several arguments. see [here](https://github.com/DevExpress/devextreme-quill/blob/master/core/quill.js#L317)

[the remark this answers is not in the document]
Comment (alexanderPolosatov) [reply modules/clipboard.js:232]: It works berause in there methods used [overload](https://github.com/DevExpress/devextreme-quill/blob/master/core/quill.js#L318) method

**Review by ksercs — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**